deploy/ansible/worker/include/run_ursula.yml

```diff
@@ -85,14 +85,14 @@
         env: "{{runtime_envvars}}"
 
     - name: "wait a few seconds for the seed node to become available"
-      when: SEED_NODE_URI is not undefined
+      when: SEED_NODE_URI is not undefined and SEED_NODE_URI
```

Comment:
is the extra check for ensuring the `SEED_NODE_URI` value is non-empty?

Reply:
yeah... it errors if SEED_NODE_URI is undefined but it can still be None.
So the first check is to avoid an error if it's undefined, the 2nd one is the actual logic of detecting if it has a non None value.